app/views/admin/dfc_product_imports/import.html.haml

```diff
@@ -3,5 +3,6 @@
 
 = render partial: 'spree/admin/shared/product_sub_menu'
 
-%p= t(".imported_products")
-= @count
+%p= t(".imported_products", count: @count)
```

Comment:
As this is an existing translation, the old translation (without count variable) is already present in about 11 locale files. This results in the `@count` variable no longer printing:

> I've noticed that on product import, the number of successfully imported products is not displayed, as it used to.

https://github.com/openfoodfoundation/openfoodnetwork/pull/13191#issuecomment-2773926148

Ideally, I think we would announce to translators to re-translate (maybe through transifex if there's a feature for that). Or create a different key.
But given that it's going to display exactly as it was before, I will go ahead and update the locale files to resolve this issue.

Reply:
Thinking about this now, I'm not sure why the spec `./spec/system/admin/dfc_product_import_spec.rb:20` did not catch this regression... It fails on my local system (before you've fixed it @dacook ).

```
  imports from given catalog (FAILED - 1)

Failures:

  1) DFC Product Import imports from given catalog
     Failure/Error: expect(page).to have_content "Imported products: 1"
       expected to find text "Imported products: 1" in "Logged in as : aliza@turner.biz Account Logout Store Dashboard Products Reports Enterprises Products Inventory Import DFC product catalog import Imported products:"
```

Reply:
Could it be due to different locales between fr_staging and the CI test environment?